### ISSUE TITLE
Overall project goal.

### ISSUE THREAD

**Issue by Thell** [Maroon502/cbc-src#1]
Hello! I just ran across your crate(s) and it looks like you are up to something exciting. Perhaps making a full blown Rust Cbc setup? I noticed that the bindings are generated using bindgen 👍 _and_ there are flags for building parallel _and_ Cbc itself is built!!!

Wow!

It looks like these are all really recent crates/commits but I'm wondering what the current state is. Is it all in a usable state to start testing and giving any feedback? Or... ?

**Comment by Maroon502** [comment]
Yeah, it's usable now. And I am just waiting for the Cbc 3.0 version to be released and update this crate. Then I will begin to start the project `cbc-rs' to provide the advanced binden. Besides, I want to start the project of ipopt and bonmin, If anyone wants to finish them with me, I'm very welcome

**Comment by Maroon502** [comment]
Actually, [coin-cbc](https://github.com/KardinalAI/coin_cbc) has provided almost all interfaces. So I haven't figured out whether to provide `CBC-RS` yet

**Comment by jacobsvante** [comment]
Thanks for these crates @Maroon502! Haven't tried them myself but they looks promising!

Being able to use CBC in Rust without installing the library manually on the system is great. It's a very finicky library to compile.

Working together with the [coin_cbc](https://github.com/KardinalAI/coin_cbc) crate project would be preferable IMO. [good_lp](https://github.com/rust-or/good_lp) uses it.
